crates/driver/src/boundary/settlement.rs

```diff
@@ -240,6 +247,37 @@ impl Settlement {
         Ok(eth::U256::from_big_rational(&quality)?.into())
     }
 
+    /// Normalized token prices.
+    pub fn prices(
```

Comment:
Good catch. Instead of moving the function to `competition::Auction`, which would require dropping `ExternalPrices` or introducing boundary for it, I just inlined the code and will add a follow up task to implement this.

Reply:
Moved into auction without any dependencies.